src/lib.rs

```diff
@@ -384,7 +384,7 @@ impl Builder {
             output_vector.push(prefix.clone());
         }
 
-        if let Some(ref prefix) = self.options.anon_fields_prefix {
+        if let prefix = &self.options.anon_fields_prefix {
```

Comment:
Sorry -- when you accepted my changes, I had not yet resolved this warning (infallible match in `if let`), which turns into an error in CI.

I am not totally sure what to do here -- if `anon_fields_prefix` is a `String` (as I made it) and not an `Option<String>` (as you made it) then the `--anon-fields-prefix` option will always be generated. That is not *necessarily* bad -- but it might be surprising.

Anyway, probably you should wait to see what @emilio thinks before you take any more of my crazy ideas :)

Reply:
Easiest one IMO is moving the default prefix to a constant, and just do:

```
if self.options.anon_fields_prefix != DEFAULT_ANON_FIELDS_PREFIX {
    output_vector.push(...);
}
```

Either that, or using `Option<>`, or using a `String` and treating the empty string as a default (as there can't be any use cases for an empty prefix because otherwise we'd generate invalid code I think) work for me.

Reply:
Hey, thanks for review :) ,it's changed to constant DEFAULT_ANON_FIELDS_PREFIX now.